lint/failure.go

```diff
@@ -81,7 +81,13 @@ type Failure struct {
 }
 
 // GetFilename returns the filename.
+// Deprecated: Use Filename.
 func (f *Failure) GetFilename() string {
+	return f.Filename()
+}
```

Comment:
If you want to respect specs, it should be in a separated paragraph.

```suggestion
 // GetFilename returns the filename.
//
// Deprecated: Use Filename.
 func (f *Failure) GetFilename() string {
	return f.Filename()
}
```

Also, you could use godoc link to help navigation


```suggestion
 // GetFilename returns the filename.
//
// Deprecated: Use [Filename].
 func (f *Failure) GetFilename() string {
	return f.Filename()
}
```